finality-aleph/src/party/manager/mod.rs

```diff
@@ -333,6 +330,11 @@ where
         };
 
         let last_block_of_previous_session = session_boundaries.first_block().saturating_sub(1);
+        let last_block_of_previous_session_hash = self
+            .client
+            .block_hash(last_block_of_previous_session)
+            .unwrap()
+            .unwrap();
```

Comment:
I'm afraid that two unwraps don't work as two minuses :(

Reply:
Changed to expect :)